src/client/components/info-bubble/info-bubble.tsx

```diff
@@ -23,7 +23,8 @@ import { SvgIcon } from "../svg-icon/svg-icon";
 import "./info-bubble.scss";
```

Comment:
You're making this component dependant on some edge case (warnings presented in header). You should make specialised version of this component via composition.